packages/@react-aria/selection/src/useSelectableItem.ts

```diff
@@ -97,7 +100,7 @@ export function useSelectableItem(options: SelectableItemOptions): SelectableIte
   }, [ref, isFocused, manager.focusedKey, manager.isFocused, shouldUseVirtualFocus]);
 
   let itemProps: SelectableItemAria['itemProps'] = {
-    tabIndex: isFocused ? 0 : -1,
+    tabIndex: isFocused && !shouldUseVirtualFocus ? 0 : -1,
```

Comment:
Makes it so hitting Tab doesn't move focus to the virtually focused combobox menu item when the combobox menu is open